modules/components/src/Aggs/AggsQuery.js

```diff
@@ -56,3 +58,9 @@ export default ({ index = '', aggs = [], sqon = null, ...props }) =>
       {...props}
     />
   );
+
```

Comment:
Could you remove this comment? 
Then this looks good 👍

Reply:
ah oops, sure thanks!